go/keybase/setup_windows.iss

```diff
@@ -50,11 +50,13 @@ Root: "HKCU"; Subkey: "SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\Keyba
 WelcomeLabel2=This will install [name/ver] on your computer.
 
 [Run]
-Filename: "{app}\keybase.exe"; Parameters: "service"; WorkingDir: "{app}"; Flags: postinstall runhidden nowait; Description: "Start Keybase service"
+Filename: "{app}\keybase.exe"; Parameters: "watchdog"; WorkingDir: "{app}"; Flags: postinstall runhidden nowait; Description: "Start Keybase service"
```

Comment:
Yes. That sounds good.

On Monday, November 9, 2015, Steve Sanders <notifications@github.com> wrote:

> In go/keybase/setup_windows.iss
> <https://github.com/keybase/client/pull/1301#discussion_r44245682>:
>
> > @@ -50,11 +50,13 @@ Root: "HKCU"; Subkey: "SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\Keyba
> >  WelcomeLabel2=This will install [name/ver] on your computer.
> >
> >  [Run]
> > -Filename: "{app}\keybase.exe"; Parameters: "service"; WorkingDir: "{app}"; Flags: postinstall runhidden nowait; Description: "Start Keybase service"
> > +Filename: "{app}\keybase.exe"; Parameters: "watchdog"; WorkingDir: "{app}"; Flags: postinstall runhidden nowait; Description: "Start Keybase service"
>
> Sure. And also I was thinking: should it be a ctl command?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1301/files#r44245682>.
>